### DISCUSSION TITLE
Carbon intensity statistics

### DISCUSSION THREAD

**Discussion by AtomicEnthusiast** [electricitymaps/electricitymaps-contrib#3646] [Ideas] [answered]
I noticed that the current carbon intensity values use 2014 statistics from the IPCC. The UNECE recently released a [Life Cycle Assessment (LCA)](https://unece.org/sites/default/files/2021-10/LCA-2.pdf) which provides newer statistics, which could be used instead.

**Comment by VIKTORVAV99** [comment, marked as answer]
This was also suggested (by me) in #4161 and the answer in there was that more direct work on #738 would begin soon and they would take these into account but changing them to the UN values is not a option at this time.

This is due to the need to communicate the changes to end customers and changing them twice so soon is something electricityMap want's to avoid.
